src/components/button-toggle/button-toggle.ts

```diff
@@ -81,7 +81,7 @@ export class MdButtonToggleGroup implements ControlValueAccessor {
 
   /** Child button toggle buttons. */
   @ContentChildren(forwardRef(() => MdButtonToggle))
-  private _buttonToggles: QueryList<MdButtonToggle> = null;
+  _buttonToggles: QueryList<MdButtonToggle> = null;
```

Comment:
`@ContentChildren` etc. also can't be private, since Angular itself needs to touch them